roles/cluster/cluster_undeploy_kepler/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- name: Grab kepler-operator source
+  command:
+    git clone https://github.com/sustainable-computing-io/kepler-operator "{{ artifact_extra_logs_dir }}/kepler"
+
+- name: Undeploy kepler-operator
+  shell:
+    cd "{{ artifact_extra_logs_dir }}/kepler"; make uninstall; make undeploy &2>1
+
+- name: Delete kepler-monitor ServiceMonitor
+  shell:
+    oc delete -n openshift-kepler-operator servicemonitor kepler-monitor --ignore-not-found
```

Comment:
you don't need `Delete kepler-monitor ServiceMonitor`, deleting the namespace will delete all of its content

but you _do_ need `Undeploy kepler-operator`, because Kepler may create Cluster-wide resources, which are not deleted when the namespace is deleted.

or `make uninstall; make undeploy` deletes the namespace it created? 🤔  not impossible (but it doesn't cost much to keep the `Delete kepler namespaces` task.

Reply:
I removed `Delete kepler-monitor ServiceMonitor` since this is definitely unneeded. The rest seems difficult to know for sure, especially since it is difficult to keep track of which resources specifically the makefile is deploying, and what resources might be added in the future.